py3status/modules/bitcoin_price.py

```diff
@@ -120,6 +110,7 @@ def post_config_hook(self):
             'YEN': '¥'
         }
         self.last_price = 0
+        self.request_timeout = 10
```

Comment:
I'm loathed to add this config parameter. I would like some more universal ones for this. Other than this I think this looks great. Currently we have to many PRs to find these nice nuggets

Reply:
At this moment, this is not a config param. I removed it earlier in first comment. We can merge this now? Maybe just give me permission to merge things for one day and I'll merge whatever I want. Heh.

I'll wait for your approval. This config parameter is trivial. I'm fine with skipping this in our documentations... and make it `10` for all... I think `bitcoin_price`'s URL is slow on fetching data too.